data-raw/wd_pcon_lad_la_rgn_ctry.R

```diff
@@ -0,0 +1,123 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Created: 05/08/2024, Cam Race
+#
+# This script creates the wd_pcon_lad_la_reg_ctry data set
+#
+# If updating the data, follow these steps:
+# 1. Load the package using `devtools::load_all(".")`
+# 2. Add in an extra year for the new year into this script within the
+#    create_timeseries_lookup function (R/datasets_utils.R)
+# 3. Check if we need to add a new year into the LAD to region lookup
```

Comment:
Again, would be good to be able to point to a specific variable to update like
`years_lad_region <- c(17:20, 22:23)`.
Also, how does someone know if they "need" to add a new year into the LAD to region lookup?

Reply:
Done